Example/PaymentSheet Example/PaymentSheet Example/PlaygroundController.swift

```diff
@@ -445,9 +445,11 @@ extension PlaygroundController {
             "use_link": settings.linkEnabled == .on,
             "use_manual_confirmation": settings.integrationType == .deferred_mc,
             "require_cvc_recollection": settings.requireCVCRecollection == .on,
-            // "supported_payment_methods": ["card", "link"], // Uncomment to override payment methods (also make sure Automatic PM's is off)
             //            "set_shipping_address": true // Uncomment to make server vend PI with shipping address populated
         ] as [String: Any]
+        if let supportedPaymentMethods = settingsToLoad.supportedPaymentMethods {
+            body["supported_payment_methods"] = supportedPaymentMethods.split(separator: ",")
```

Comment:
without this, the payment sheet backend returns types like US Bank which means Instant Debits will not be supported

Reply:
any need/value to truncate the values in the array in case they included spaces between the methods or will the backend take care of that?